Examples/Tests/Macroscopic_Maxwell/inputs_3d_PR10

```diff
@@ -0,0 +1,111 @@
+my_constants.TP = 1.5385e-9 # Gaussian pulse width
+my_constants.flag_none = 0
+my_constants.flag_hs = 1
+my_constants.flag_ss = 2
+
+#################################
+############ NUMERICS ###########
+#################################
+warpx.verbose = 0
+warpx.use_filter = 0
+warpx.cfl = 0.9
+warpx.do_pml = 0
+#warpx.mag_time_scheme_order = 2 # default 1
+#warpx.mag_M_normalization = 1 # 1 is saturated
+#warpx.mag_LLG_coupling = 1
+particles.nspecies = 0
+
+algo.em_solver_medium = macroscopic # vacuum/macroscopic
+algo.macroscopic_sigma_method = laxwendroff # laxwendroff or backwardeuler
+
+macroscopic.sigma_init_style = "parse_sigma_function" # parse or "constant"
+macroscopic.sigma_function(x,y,z) = "0.0"
+macroscopic.epsilon_init_style = "parse_epsilon_function" # parse or "constant"
+macroscopic.epsilon_function(x,y,z) = "8.8541878128e-12"
+macroscopic.mu_init_style = "parse_mu_function" # parse or "constant"
+macroscopic.mu_function(x,y,z) = "1.1310e-05" # mu_r = 9
+
+#unit conversion: 1 Gauss = (1000/4pi) A/m
+#macroscopic.mag_Ms_init_style = "parse_mag_Ms_function" # parse or "constant"
+#macroscopic.mag_Ms_function(x,y,z) = "0" # in unit A/m, equal to 1750 Gauss; Ms must be nonzero for LLG
+#macroscopic.mag_alpha_init_style = "parse_mag_alpha_function" # parse or "constant"
+#macroscopic.mag_alpha_function(x,y,z) = "0.0058" # alpha is unitless, calculated from linewidth Delta_H = 40 Oersted
+#macroscopic.mag_gamma_init_style = "parse_mag_gamma_function" # parse or "constant"
+#macroscopic.mag_gamma_function(x,y,z) = "-1.759e11" # gyromagnetic ratio is constant for electrons in all materials
+
+#macroscopic.mag_max_iter = 100 # maximum number of M iteration in each time step
+#macroscopic.mag_tol = 1.e-6 # M magnitude relative error tolerance compared to previous iteration
+#macroscopic.mag_normalized_error = 0.1 # if M magnitude relatively changes more than this value, raise a red flag
+
+#################################
+############ FIELDS #############
+#################################
+warpx.E_ext_grid_init_style = parse_E_ext_grid_function
+warpx.Ex_external_grid_function(x,y,z) = 0.
+warpx.Ey_external_grid_function(x,y,z) = 0.
+warpx.Ez_external_grid_function(x,y,z) = 0.
+
+warpx.E_excitation_on_grid_style = "parse_E_excitation_grid_function"
+warpx.Ex_excitation_grid_function(x,y,z,t) = "0.0"
+warpx.Ey_excitation_grid_function(x,y,z,t) = "1000*(exp(-(t-3*TP)**2/(2*TP**2))*cos(2*pi*c/wavelength*t)) * (z>=h) * (z<h+6.0e-4)"
+warpx.Ez_excitation_grid_function(x,y,z,t) = "0.0"
+warpx.Ex_excitation_flag_function(x,y,z) = "flag_none"
+warpx.Ey_excitation_flag_function(x,y,z) = "flag_hs * (z>=h) * (z<h+6.0e-4) + flag_none * (z<h) * (z>=h+6.0e-4)"
+warpx.Ez_excitation_flag_function(x,y,z) = "flag_none"
+
+warpx.B_ext_grid_init_style = parse_B_ext_grid_function
+warpx.Bx_external_grid_function(x,y,z)= "0.0"
+warpx.By_external_grid_function(x,y,z)= 0.
+warpx.Bz_external_grid_function(x,y,z) = 0.
+
```

Comment:
For these USE_LLG=FALSE inputs files, we should delete the H and M-related input parameters to not mislead (even if they are commented out).  This includes the fields_to_plot line below.  And the mag_alpha/gamma stuff too.

Reply:
agreed. The H, M, alpha and Ms part has been deleted. Thank you!